Detectors/FIT/FT0/simulation/include/FT0Simulation/Digits2Raw.h

```diff
@@ -48,8 +49,8 @@ class Digits2Raw
 
  public:
   Digits2Raw() = default;
-  Digits2Raw(const std::string fileRaw, std::string fileDigitsName);
-  void readDigits(const std::string fileDigitsName);
+  Digits2Raw(const std::string fileRawName, const std::string fileDigitsName);
+  void readDigits(const std::string fileRawName, const std::string fileDigitsName);
```

Comment:
you still create a copy of the strings instead of passing the reference, use e.g.
`void readDigits(const std::string &fileRawName, const std::string &fileDigitsName);```